scripts/Project.toml

```diff
@@ -0,0 +1,2 @@
+[deps]
```

Comment:
Maybe a bound on Clang version?

Reply:
I'm not sure there is much to be gained given how infrequently the code is run. 

If Clang.jl changes in the future, I'd rather it broke so that someone could investigate why, rather than re-running the wrapper with an old version of the package that may not support the latest Julia syntax.